iroh-resolver/src/unixfs.rs

```diff
@@ -52,6 +54,13 @@ pub struct Link {
     pub tsize: Option<u64>,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct LinkRef<'a> {
+    pub cid: Cid,
+    pub name: Option<&'a str>,
+    pub tsize: Option<u64>,
+}
```

Comment:
Do we want `LinkRef` & `Link`, because `LinkRef` makes it easier for us to compare `name` fields? Guess I'm trying to understand why we have both.

Reply:
I wantLinkRef because it doesn't need to allocate for the name, keep Link around for the moment, as this is still evolving